src/api/api.ts

```diff
@@ -10,9 +10,14 @@ function buildRequester(method: Method) {
     return async function requester<T>(
         url: string,
         idToken: string,
-        config?: { etag?: string; data?: AxiosRequestConfig["data"] }
+        config?: {
+            etag?: string;
+            data?: AxiosRequestConfig["data"];
+            responseType?: AxiosRequestConfig["responseType"];
```

Comment:
To download excel/other binary files, we need to be able to set `responseType: 'arraybuffer'`.